src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -735,6 +735,23 @@ public function testKernelWithoutBundles()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation The Symfony\Component\HttpKernel\Kernel::getEnvParameters() method is deprecated as of 3.3 and will be removed in 4.0. Use the %s syntax to get the value of any environment variable from configuration files instead.
+     * @expectedDeprecation The support of special environment variables that start with SYMFONY__ (such as "SYMFONY__FOO__BAR") is deprecated as of 3.3 and will be removed in 4.0. Use the %env()% syntax instead to get the value of environment variables in configuration files.
```

Comment:
The use of `%env()%` makes the assertion fails due to the use of [`assertStringMatchesFormat`](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php#L258). For now I didn't find better than replacing `%env()%` by `%s` in the expected deprecation to solve this issue